htdocs/admin/company.php

```diff
@@ -137,7 +137,7 @@
 	$arrayofimages = array('logo', 'logo_squarred');
 	//var_dump($_FILES); exit;
 	foreach ($arrayofimages as $varforimage) {
-		if ($_FILES[$varforimage]["name"] && !preg_match('/(\.jpeg|\.jpg|\.png)$/i', $_FILES[$varforimage]["name"])) {	// Logo can be used on a lot of different places. Only jpg and png can be supported.
+		if ($_FILES[$varforimage]["name"] && !preg_match('/(\.jpeg|\.jpg|\.png'.($conf->global->MAIN_ALLOW_SVG_FILES_AS_IMAGES?"|\.svg":"").')$/i', $_FILES[$varforimage]["name"])) {	// Logo can be used on a lot of different places. Only jpg and png can be supported.
```

Comment:
The test preg_match should be replaced with a test 
`image_format_supported(_FILES[$varforimage]["name"], 1)`